tests/Feature/MessageTest.php

```diff
@@ -42,4 +46,42 @@ public function test_message_is_not_found()
 
         $response->assertNotFound();
     }
+
+    public function test_click_button_and_add_like()
+    {
+        $user = User::factory()->create();
+        $this->actingAs($user);
+        $message = Message::factory()->create();
+
+        $livewire = Livewire::test(LikeButton::class, ['message' => $message]);
+        $livewire->call('onClick');
+        $livewire->assertSet('likeCount', 1);
+        $livewire->assertSet('icon', '❤️');
```

Comment:
(nits)
icon は定数で定義したほうが変更に強いし、コードがわかりやすくなるかと思います。
```suggestion
        $livewire->assertSet('icon', LikeButton::LIKED);
```